go/client/cmd_id.go

```diff
@@ -89,6 +92,11 @@ func NewCmdID(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 				Usage:     "Use identify UI.",
 				HideUsage: !develUsage,
 			},
+			cli.BoolFlag{
+				Name:      "s, skip-proof-cache",
+				Usage:     "Skip cached proofs, force re-check",
+				HideUsage: !develUsage,
```

Comment:
hey I just realized, I copied this from another command - is this supposed to be only for devel?